_translator-files/po/de/Installation-for-Android.po

```diff
@@ -41,7 +43,7 @@ msgstr "Installation auf Android"
 #. type: Plain text
 #: ../wiki/en/Installation-for-Android.md:9
 msgid "{% include breadcrumb.html root=\"Using Jamulus\" branch1=\"Getting Started\" branch1-url=\"Getting-Started\" %}"
-msgstr "{% include breadcrumb.html root=\"Jamulus Nutzen\" branch1=\"Erste Schritte\" branch1-url=\"Getting-Started\" %}"
+msgstr "{% include breadcrumb.html root=\"Using Jamulus\" branch1=\"Getting Started\" branch1-url=\"Getting-Started\" %}"
```

Comment:
In `"Using Jamulus\" branch1=\"Getting Started\" branch1-url=\"Getting-Started\"`, the 'branches' should be translated. Not the url (Getting-Started) at the end.

Reply:
Noted and translated